test/distributed/tensor/test_utils.py

```diff
@@ -102,6 +103,37 @@ def test_explicit_order_placements(self):
                     test_case["mesh_shape"], test_case["placements"]
                 )
 
+    def test_compute_local_shape_and_global_offset_uneven(self):
+        # This case is not only 'uneven' bug also has an empty shard
```

Comment:
need clean up the test:
- remove print
- change the if-else block to
```
exp_local_shape = ...
local_shape = ...
self.assertEqual(local_shape, exp_local_shape)
self.assertEqual(global_offset, exp_global_offset)
```

Reply:
cleaned up.